doc/docs/master/reference/pipeline_spec.md

```diff
@@ -192,14 +179,61 @@ create pipeline](pachctl/pachctl_create_pipeline.md) section.
        "branch": string,
        "glob": string,
        "join_on": string,
-      "outer_join": bool,
+       "outer_join": bool,
+       "lazy": bool,
+       "empty_files": bool,
+       "s3": bool
+    }
+  }
+]
+
+
+------------------------------------
+"group" input
+------------------------------------
+
+"group": [
+  {
+    "pfs": {
+      "name": string,
+      "repo": string,
+      "branch": string,
+      "glob": string,
+      "group_by": string,
+      "lazy": bool,
+      "empty_files": bool,
+      "s3": bool
+    }
+  },
+  {
+    "pfs": {
+       "name": string,
+       "repo": string,
+       "branch": string,
+       "glob": string,
+       "group_by": string,
        "lazy": bool,
        "empty_files": bool,
        "s3": bool
     }
   }
 ]
 
+
+
+------------------------------------
+"cron" input
+------------------------------------
+
+"cron": {
+    "name": string,
+    "spec": string,
+    "repo": string,
+    "start": time,
+    "overwrite": bool
+}
+
+
 ------------------------------------
 "git" input
```

Comment:
git inputs will be deprecated soon (for build pipelines). Let's check with eng and figure out where/when to add deprecation notices to both the docs and product.

Reply:
Noted.